src/applications/check-in/components/AppointmentBlockVaos.jsx

```diff
@@ -24,7 +24,10 @@ const AppointmentBlockVaos = props => {
   const handleDetailClick = (e, appointment) => {
     e.preventDefault();
     recordEvent({
-      event: createAnalyticsSlug('details-link-clicked', 'nav'),
+      event: createAnalyticsSlug(
+        'details-link-clicked',
+        `nav${app === 'preCheckIn' ? '-pre' : ''}`,
```

Comment:
should we add an app param to createAnalyticsSlug instead of this? Could be a follow on ticket, but would feel nicer I think.